### DISCUSSION TITLE
Creating a standard field schema for the forms

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#5] [Ideas]
In the current spec, each lender can have their own forms with different fields.
The same field can be defined differently by different lenders as well.
For example:
Lender 1 - PAN
Lender 2 - Permanent Account Number
Lender 3 - Permanent Account No.

I think we should standardise the field names, otherwise there could be a situation that either the BAPs are not filling up the fields correctly or not provide the data at all, resulting in inadequate applications and rejections.

**Comment by Antriksh-Parmar** [comment]
Yes. @92shreyansh could you pls respond?

**Comment by 92shreyansh** [comment]
Hi @ankitagarwaladobe,

The current form illustartion in the examples include in the draft we have standardized the id in the form, BPP can use the field text to there descreation keeping the id consistant to the specification.

Lender 1 - `<label for="pan">PAN</label><input type="text" id="pan" name="pan" required />`
Lender 2 - `<label for="pan">Permanent Account Number</label><input type="text" id="pan" name="pan" required />`
Lender 3 - `<label for="pan">Permanent Account No.</label><input type="text" id="pan" name="pan" required />`

@Antriksh-Parmar & @ankitagarwaladobe Feel free to raise a PR in case any update or addition of ids is required in the forms.

**Reply by ankitagarwaladobe** [reply]
Thanks @92shreyansh 
SGTM

I think we need to callout that the BPPs have to use the same ids.

**Reply by ankitagarwaladobe** [reply]
@92shreyansh 
I think we need to also standardise the field type in the spec. Like all of them could be text rather than few being numbers/text etc.

**Reply by 92shreyansh** [reply]
Yes, Agreed we would update the same. We are working on lisiting these field details in the specifications similar / along side attribute UI with deatils of standard id, required/optional, type, & description
